frontend/Pages/Login.razor

```diff
@@ -1,19 +1,35 @@
+@using MudBlazor
+@using System.ComponentModel.DataAnnotations
+@inject ISnackbar Snackbar

```

Comment:
We inject snackbar, but we are not using it?

Reply:
ye who cares